spark-k8s/CHANGELOG.md

```diff
@@ -18,8 +18,9 @@
 
 ### Changed
 
-- Base image changed to `stackable/image/vector` ([#342]).
```

Comment:
I'm only on my phone and might misread the diff but why did you remove this? The reference still seems to be there

Reply:
It's because the folder has been renamed from pyspark to spark.
But in fact, I would like to get rid of the individual changelog files because:
* they are not maintained 
* they cannot be automatically updated in the release process

Instead, all changes in this repo should be documented in the root changelog file.

Reply:
I updated the changelog with the old spark entries so nothing is lost for now: https://github.com/stackabletech/docker-images/pull/436/commits/87b2063f794aa4d231a4c8d2d15184576d3ae8e8